C/common/bearer_token.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Fledge bearer token utilities
+ *
+ * Copyright (c) 2021 Dianomic Systems
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Massimiliano Pinto
+ */
+
+#include "bearer_token.h"
+
+using namespace std;
+using HttpServer = SimpleWeb::Server<SimpleWeb::HTTP>;
+
```

Comment:
I don't really like these free floating C functions and the use of the vector. It would have been better to have a class to represent the bearer token, the constructor would be the thing to extract it from the request, etc.